### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by yoloking1111** [UncreatedStaff/UncreatedWarfare#126]
im asking for c4 or other bombs to be able to be picked back up after they are placed, because you might place it somewhere accidently and you will need to destroy it or remotely blow it up and might damage teammates.

**Comment by UsernameIsUnavailible** [comment]
no, This got disabled to avoid people duping their C4/Landmines.
